variables.tf

```
@@ -80,3 +80,9 @@ variable "nat_gateway_enabled" {
   description = "Flag to enable/disable NAT gateways for private subnets"
   default     = "true"
 }
+
+variable "auto_assign_public_ip" {
```

```suggestion
variable "map_public_ip_on_launch" {
```

Let's use the same name as Terraform does, for consistency

---

Agreed